docs/modules/development/pages/osgi/osgi-and-opennms.adoc

```diff
@@ -0,0 +1,135 @@
+[[OSGI-and-OpenNMS]]
+= OSGI Integration within OpenNMS
+
```

Comment:
Could we add a short introduction here? Something like "OSGi integration lets you do x, y, z."
Do we need to define OSGi, or will readers know what it is?

Reply:
Should be covered by https://github.com/OpenNMS/opennms/pull/4614/commits/9b1d2d8444d1c0c5558fba0ac97087785d13e2cb